src/cmdlib.sh

```diff
@@ -477,20 +480,29 @@ EOF
         srcvirtfs=("-virtfs" "local,id=source,path=${workdir}/src/config,security_model=none,mount_tag=source")
     fi
 
-    rm -f "${workdir}/tmp/rc"
+    local runvm_console
+    runvm_console="${workdir}/tmp/runvm-console.txt"
+    rm -f "${workdir}/tmp/rc" "${runvm_console}" "${cmdoutput}"
 
     #shellcheck disable=SC2086
-    kola qemuexec -m 2048 --auto-cpus -U --workdir none -- -no-reboot -nodefaults -serial stdio \
+    touch "${runvm_console}"
+    tail -qF "${cmdoutput}" &
```

Comment:
Do we also want `--pid $$` here?

Reply:
Added that and our friend `setpriv --pdeathsig SIGTERM` too.